### PR TITLE
Update ParticipantTable page

### LINKED ISSUE #134: Improve responsiveness of newdle summary page
image

### PR DESCRIPTION
**Pull request by hugonxc**
Hey there, I got some troubles with my branchs. So I decided reopen.

This is my final version of correction of responsiveness:
desk_after_none
desk_after_user
mobile_after_none
mobile_after_user

I just have a question about the migration file. Should I deleted or add it to gitignore.

closes #134

### CODE REVIEW

**Comment by ThiefMaster** [comment]
> I just have a question about the migration file. Should I deleted or add it to gitignore.

Neither. You should not even run `make newdb` since it's meant for cases where someone changes the database structure while we are still in heavy-development mode (and thus do not want revisions).
So the solution here is to remove the changes to that file altogether:

The easiest way to do this is:

```sh
git reset 'HEAD^'
git add newdle/migrations/versions/20191025_1543_000000000000_create_initial_tables.py
git rm newdle/migrations/versions/20190912_1718_000000000000_create_initial_tables.py
git add newdle/client/src/components/ParticipantTable.*
git commit
git push -f
```

**Comment by ThiefMaster** [comment]
Since you replaced the old commit, please keep the commit message describing the original change. You can update it now using `git commit --amend` followed by `git push -f`

**Comment by hugonxc** [comment]
> Neither. You should not even run `make newdb` since it's meant for cases where someone changes the database structure while we are still in heavy-development mode (and thus do not want revisions).

Ok, I think it's a great to have this more clearly on README

**Comment by ThiefMaster** [comment]
> Ok, I think it's a great to have this more clearly on README

To be honest, we didn't expect so much external contributors when we added this ;) 
FWIW, the readme does mention what `make newdb` does, and doesn't ask you to run it. Anyway, I think we can soon remove it since the DB structure is kind of stable...

**Comment by ThiefMaster** [comment]
Unfortunately this page looks pretty broken with your current changes:

image

image

**Comment by hugonxc** [comment]
> git add newdle/migrations/versions/20191025_1543_000000000000_create_initial_tables.py
This file doesn't no more.
> git rm newdle/migrations/versions/20190912_1718_000000000000_create_initial_tables.py
By looking on the master I found this...

**Comment by ThiefMaster** [comment]
OK, maybe it's easier if you you `git reset HEAD^` and then use `git add -p` to only stage your actual changes for committing. Then commit them, and after committing throw all uncommitted changes away using `git reset --hard`

**Comment by hugonxc** [comment]
> image

I think this could be triggered by browser switching to different modes.

> 
> image

It's better if align left?

**Comment by ThiefMaster** [comment]
Yes, the date/time column should only use as much space as it needs. Also, the circles should be left-aligned (like it was before).

File: newdle/client/src/components/ParticipantTable.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ function AvailabilityRow({availability: {startDt, available}, setActiveDate, act
       onClick={() => setActiveDate(startDt)}
       active={active}
     >
-      <Table.Cell width={3}>
+      <Table.Cell width={200} className={styles['schedule-info']}>
```
Comment (mic4ael) [inline newdle/client/src/components/ParticipantTable.js]: the value you passed as the `width` prop is invalid (https://react.semantic-ui.com/collections/table/)

**Comment by mic4ael** [comment]
I quickly had a  look into that and here is a diff what I have done to make it more responsive. Maybe you could have a look at this and improve it in case you find some problems?

```diff
diff --git i/newdle/client/src/components/ParticipantTable.js w/newdle/client/src/components/ParticipantTable.js
index ec22cbe..cde1387 100644
--- i/newdle/client/src/components/ParticipantTable.js
+++ w/newdle/client/src/components/ParticipantTable.js
@@ -110,7 +110,7 @@ export default function ParticipantTable({finalDate, setFinalDate}) {
 
   return (
     <div className={styles['participant-table']}>
-      <Table textAlign="center" definition selectable>
+      <Table textAlign="center" definition selectable unstackable>
         <Table.Body>
           {availabilityData.map(availability => (
             <AvailabilityRow
diff --git i/newdle/client/src/components/ParticipantTable.module.scss w/newdle/client/src/components/ParticipantTable.module.scss
index 3e9e17c..9482c5c 100644
--- i/newdle/client/src/components/ParticipantTable.module.scss
+++ w/newdle/client/src/components/ParticipantTable.module.scss
@@ -38,6 +38,10 @@
         display: flex;
         align-items: center;
 
+        @media (max-width: 600px) {
+          flex-direction: column;
+        }
+
         .availability-indicator {
           flex: 1;
           margin-right: 20px;
```

**Comment by mic4ael** [comment]
Hey @hugonxc, are you still working on this?

**Pull request closed without merging**